content/docs/lists-and-keys.md

```diff
@@ -109,7 +109,7 @@ const listItems = numbers.map((number) =>
 );
 ```
 
-The best way to pick a key is to use a string that uniquely identifies a list item among its siblings. Most often you would use IDs from your data as keys:
+La mejor forma de elegir una key es usar un string que idenfique unicamente a un elemento de la lista entre sus hermanos. Con mas frecuencia vas a usar IDs de tus datos como key:
```

Comment:
Maybe `.. es usar un string ...` can be changed to `... es usando un string ...`

`únicamente`?

For the last sentence, this is my sugestion:
> A menudo usarías IDs de tus datos como key: